### PR TITLE
Add tests for abt

### PR DESCRIPTION
**Pull request by radare**
**Your checklist for this pull request**
- [ ] I've read the [guidelines for contributing](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md) to this repository
- [ ] I made sure to follow the project's [coding style](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [radare2 book](https://github.com/radareorg/radare2book) with the relevant information (if needed)

**Detailed description**

...

**Test plan**

...

**Closing issues**

...

### CODE REVIEW

File: test/new/db/anal/path [hunk 1]
```diff
@@ -12,7 +12,6 @@ EXPECT=<<EOF
 0x1000012c6
 0x1000012e7
 0x100001682
-0x10000168b
```
Comment (radare) [inline test/new/db/anal/path]: Thats wrong.
1) The last bb should be also listed
2) an instruction in the middle of a bb should be a valid argument

File: test/new/db/anal/path [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+NAME=abt2
+FILE=../bins/mach0/mac-ls2
+BROKEN=1
+CMDS=<<EOF
+aaa
+abt 0x1000015bf @ 0x1000011e8
+EOF
+EXPECT=<<EOF
+XXX
```
Comment (radare) [inline test/new/db/anal/path]: This test should pass

File: libr/core/cmd_anal.c [hunk 3]
```diff
@@ -8864,6 +8864,21 @@ static void cmd_anal_abt(RCore *core, const char *input) {
 			n = *(++p)? r_num_math (core->num, p): 1;
 		}
 		addr = r_num_math (core->num, input + 1);
+
+#if 1
```
Comment (radare) [inline libr/core/cmd_anal.c]: Remove if block

File: test/new/db/anal/path (hunk 1 above)
Comment (thestr4ng3r) [reply test/new/db/anal/path]: 1 is trivial.
I explained on telegram why 2 is bad.

**Pull request merged**